tests/integration/cli/tests/fixtures/init3.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "wasmer"
+version = "0.5.6"
+description = "hello"
+
+[dependencies]
+
+[package.metadata.wapm]
```

Comment:
This file shouldn't exist is a `cargo wapm` one which we agreed not to have in this PR

Reply:
Note: unless is used for testing the `cargo.toml` naming parsing into `wasmer init`

Reply:
I removed `init3` and `init5`, we don't need them anymore.